Cargo.toml

```diff
@@ -18,12 +18,17 @@ maintenance = { status = "actively-developed" }
 [dependencies]
 anyhow = "1.0"
 byteorder = "1.5"
+bzip2 = { version = "0.4", optional = true }
```

Comment:
I assume packaged versions are compatible with this one, right?

`Version        : 0.5.2`